### PR TITLE
Update color styling

### PR DESCRIPTION
**Pull request by Akswii**
Updated colors to look like they did in experimental.

### CODE REVIEW

File: src/App/Documentation/core/Color/index.js [hunk 1]
```diff
@@ -1,137 +1,149 @@
...
-                <div className="col-xl col-lg-2 col-12 mb-lg-0">
-                    <ColorPreview name="blue-bright-alt" hex="#a3c9cf" />
-                </div>
-                <div className="col-xl col-lg-2 col-12 mb-lg-0">
-                    <ColorPreview name="red-medium" hex="#dd3300" />
-                </div>
-            </div>
-        </ComponentPreview>
+        <div className="row">
+            <div className="col- 12 col-sm mb-lg-0">
```
Comment (hmosvoll) [inline src/App/Documentation/core/Color/index.js]: Should it be a space here?

File: src/App/Documentation/core/Color/index.js [hunk 2]
```diff
@@ -1,137 +1,149 @@
...
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="nearwhite" hex="#f0f0f0" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="white" hex="#ffffff" color="black" />
-                </div>
-            </div>
-        </ComponentPreview>
+        <div className="row mt-3">
```
Comment (hmosvoll) [inline src/App/Documentation/core/Color/index.js]: More of a general comment. We should start noticing where we use these vertical spacing classes, for future spacing adjustments.

File: src/App/Documentation/core/Color/index.js (hunk 1 above)
Comment (Akswii) [reply src/App/Documentation/core/Color/index.js]: Nope, good catch 💯

**Comment by Akswii** [comment]
> 
> 
> A lot of divs here, any idea how you could do it more semantic?

When using our version of bootstrap grid, it has a tendency to require a lot of divs. A solution could be to add grid behavior through mixins for ".color-preview". Found [this](https://medium.com/@erik_flowers/how-youve-been-getting-the-bootstrap-grid-all-wrong-and-how-to-fix-it-6d97b920aa40) when googling.

File: src/App/Documentation/core/Color/index.js (hunk 2 above)
Comment (Akswii) [reply src/App/Documentation/core/Color/index.js]: I'm not using `.showcase-panel` for colors, so there is no box with margin and padding separating it from the heading above. I added "margin-top" to give it more space. All of our documentation will have to be updated with new margin/padding if we stop using `.showcase-panel`.

**Comment by hmosvoll** [comment]
> > A lot of divs here, any idea how you could do it more semantic?
> 
> When using our version of bootstrap grid, it has a tendency to require a lot of divs. A solution could be to add grid behavior through mixins for ".color-preview". Found [this](https://medium.com/@erik_flowers/how-youve-been-getting-the-bootstrap-grid-all-wrong-and-how-to-fix-it-6d97b920aa40) when googling.

What about something like this?
https://jsfiddle.net/dusak7mt/

**Comment by Akswii** [comment]
> 
> 
> > > A lot of divs here, any idea how you could do it more semantic?
> > 
> > 
> > When using our version of bootstrap grid, it has a tendency to require a lot of divs. A solution could be to add grid behavior through mixins for ".color-preview". Found [this](https://medium.com/@erik_flowers/how-youve-been-getting-the-bootstrap-grid-all-wrong-and-how-to-fix-it-6d97b920aa40) when googling.
> 
> Something like this?
> https://jsfiddle.net/dusak7mt/

Yes, but I would need a parent class to set styling on the child tags, something on <ul>. Maybe .doc-color-list or something.

**Comment by hmosvoll** [comment]
> > > > A lot of divs here, any idea how you could do it more semantic?
> > > 
> > > 
> > > When using our version of bootstrap grid, it has a tendency to require a lot of divs. A solution could be to add grid behavior through mixins for ".color-preview". Found [this](https://medium.com/@erik_flowers/how-youve-been-getting-the-bootstrap-grid-all-wrong-and-how-to-fix-it-6d97b920aa40) when googling.
> > 
> > 
> > Something like this?
> > https://jsfiddle.net/dusak7mt/
> 
> Yes, but I would need a parent class to set styling on the child tags, something on `<ul>`. Maybe .doc-color-list or something.

Yea, the example mainly focuses on the HTML structure, the CSS selectors needs some changes to work in a real world scenario.

**Comment by Akswii** [comment]
> 
> 
> > > > > A lot of divs here, any idea how you could do it more semantic?
> > > > 
> > > > 
> > > > When using our version of bootstrap grid, it has a tendency to require a lot of divs. A solution could be to add grid behavior through mixins for ".color-preview". Found [this](https://medium.com/@erik_flowers/how-youve-been-getting-the-bootstrap-grid-all-wrong-and-how-to-fix-it-6d97b920aa40) when googling.
> > > 
> > > 
> > > Something like this?
> > > https://jsfiddle.net/dusak7mt/
> > 
> > 
> > Yes, but I would need a parent class to set styling on the child tags, something on `<ul>`. Maybe .doc-color-list or something.
> 
> Yea, the example mainly focuses on the HTML structure, the CSS selectors needs some changes to work in a real world scenario.

So it is still OK to have `.row` and `.col` on the HTML tags? Or do you wish for the grid styling to be applied directly to the tag?

**Comment by hmosvoll** [comment]
> > > > > > A lot of divs here, any idea how you could do it more semantic?
> > > > > 
> > > > > 
> > > > > When using our version of bootstrap grid, it has a tendency to require a lot of divs. A solution could be to add grid behavior through mixins for ".color-preview". Found [this](https://medium.com/@erik_flowers/how-youve-been-getting-the-bootstrap-grid-all-wrong-and-how-to-fix-it-6d97b920aa40) when googling.
> > > > 
> > > > 
> > > > Something like this?
> > > > https://jsfiddle.net/dusak7mt/
> > > 
> > > 
> > > Yes, but I would need a parent class to set styling on the child tags, something on `<ul>`. Maybe .doc-color-list or something.
> > 
> > 
> > Yea, the example mainly focuses on the HTML structure, the CSS selectors needs some changes to work in a real world scenario.
> 
> So it is still OK to have `.row` and `.col` on the HTML tags? Or do you wish for the grid styling to be applied directly to the tag?

How and why would use the  `.row` and `.col`?

File: src/less/designguide.less [hunk 3]
```diff
@@ -65,32 +65,25 @@
...
-                margin-bottom: 0.5rem;
-            }
+        figcaption {
+            line-height: 20px;
+            margin-top: 5px;
         }
+    }
 
+    @media screen and (min-width: @screen-sm-min) {
         .color-preview {
```
Comment (hmosvoll) [inline src/less/designguide.less]: Any reason why dropped the max width idea?

File: src/less/designguide.less [hunk 4]
```diff
@@ -65,32 +65,25 @@
         font-size: 87.5%;
     }
 
-    .color-container {
-        display: flex;
-        justify-content: center;
+    .color-preview {
+        display: grid;
+        gap: 20px;
```
Comment (hmosvoll) [inline src/less/designguide.less]: What do you think about doubling the gap? Any reason for the change over to using a lot of px instead of rem?

**Pull request merged**